salt/reactors/vault/alert_expiring_leases.sls

```diff
@@ -0,0 +1,12 @@
+alert_on_lease_near_expiration:
+  local.slack.post_message:
+    - tgt: 'roles:master'
+    - expr_form: grain
+    - kwarg:
+        channel: "#devops"
+        message: |
+          @channel The Vault lease `{{ data['id'] }}` will be expiring at `{{ data['expire_time'] }}`.
```

Comment:
No quotes needed here?

Reply:
The quotes aren't needed because it is being rendered as multi-line YAML